attest/untrusted/Cargo.toml

```
@@ -17,7 +17,8 @@ cfg-if = "1.0"
 displaydoc = { version = "0.2", default-features = false }
 mc-attest-core = { path = "../core", default-features = false }
 mc-attest-verifier = { path = "../verifier", default-features = false }
-mc-attestation-verifier = "0.3.1"
+mc-attest-verifier-types = { path = "../verifier/types", default-features = false }
+mc-attestation-verifier = "0.3.0"
```

:wrench: Should probably leave the `mc-attestation-verifier` at 0.3.1
```suggestion
mc-attestation-verifier = "0.3.1"
```
Updating this and the other one should thin out the Cargo.lock changes